languages/rust/exercises/concept/structs/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#[derive(Clone, Debug)]
+pub struct User {
+    name: String,
+    age: u32,
+    weight: f32,
+}
+
+impl User {
+    pub fn new(name: String, age: u32, weight: f32) -> Self {
+        unimplemented!()
+    }
+
+    pub fn get_name(&self) -> &str {
```

Comment:
Idiomatic getters have the same name as the field they're getting.

```suggestion
    pub fn name(&self) -> &str {
```